octobot_trading/exchanges/types/rest_exchange.py

```diff
@@ -163,6 +163,45 @@ async def _edit_order(self, order_id: str, order_type: enums.TraderOrderType, sy
                                                quantity, price, stop_price, side,
                                                current_price, params)
 
+    async def edit_order_by_replacing(
```

Comment:
Can you add a test for it please?

Reply:
Not sure how to test that, ss there somewhere a similar example test for a real exchange trading test?

Reply:
A test like https://github.com/Drakkar-Software/OctoBot-Trading/blob/master/tests/exchanges/implementations/test_default_rest_exchange.py#L46 to ensure that the new order is properly createdand to also test that the portfolio values are properly updated (available and total)